tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -546,3 +549,103 @@ def test_positive_virt_card(session, target_sat, module_location, module_org, vm
+    get_vmware_datastore_summary_string,
+):
+    """Assign Ansible Role to a Hostgroup and verify after provision a Host, job is schedule
+    for ansible role
+
+    :id: 500f32e8-c1db-4ef9-ae20-0dbb5bccf2ea
+
+    :steps:
+        1. Import role(s) available by default.
+        2. Assign role(s) to Hostgroup.
+        3. Provision a Host with Hostgroup.
+        4. In Host -> Overview -> Select the "Recent Job" action.
+        5. Click the Schedule Job button.
+        6. Verify that the Ansible roles has been Scheduled.
+
+    :expectedresults: The role is successfully scheduled with provisioned host.
+
+    :BZ: 2025523
+
+    :customerscenario: true
+    """
+    SELECTED_ROLE = 'theforeman.foreman_scap_client'
+    host_name = gen_string('alpha').lower()
+    guest_os_names = 'Red Hat Enterprise Linux 8 (64 bit)'
+    storage_data = {'storage': {'disks': [{'data_store': get_vmware_datastore_summary_string}]}}
+    network_data = {
+        'network_interfaces': {
+            'nic_type': VMWARE_CONSTANTS['network_interface_name'],
+            'network': f'VLAN {settings.provisioning.vlan_id}',
+        }
+    }
+    target_sat.cli.Settings.set({'name': "remote_execution_connect_by_ip", 'value': "yes"})
+    with target_sat.ui_session() as session:
+        session.ansibleroles.import_all_roles()
+        session.location.select(module_location.name)
+        session.organization.select(module_sca_manifest_org.name)
+        assert session.ansibleroles.import_all_roles() == session.ansibleroles.imported_roles_count
+        session.hostgroup.assign_role_to_hostgroup(
+            module_vmware_hostgroup.name, {'ansible_roles.resources': SELECTED_ROLE}
+        )
+        assert SELECTED_ROLE in session.hostgroup.read_role(
+            module_vmware_hostgroup.name, SELECTED_ROLE
+        )
+        session.computeresource.update_computeprofile(
+            entity_name=module_vmware_cr.name,
+            compute_profile=COMPUTE_PROFILE_SMALL,
+            values={
+                'provider_content.memory': '6000',
+                'provider_content.cluster': settings.vmware.cluster,
+                'provider_content.guest_os': guest_os_names,
+                'provider_content.storage': [value for value in storage_data.values()],
+                'provider_content.network_interfaces': [value for value in network_data.values()],
+            },
+        )
+        session.host.create(
+            {
+                'host.name': host_name,
+                'host.hostgroup': module_vmware_hostgroup.name,
+                'host.inherit_deploy_option': False,
+                'host.deploy': module_vmware_cr.name + f' ({FOREMAN_PROVIDERS["vmware"]})',
+                'host.inherit_compute_profile_option': False,
+                'host.compute_profile': COMPUTE_PROFILE_SMALL,
+                'host.content_source': target_sat.hostname,
+                'host.lce': ENVIRONMENT,
+                'host.content_view': DEFAULT_CV,
```

Comment:
module_vmware_hostgroup fixture already has this field selected then we can skip this while host create